static/docs/commands-reference/destroy.md

```diff
@@ -39,31 +39,33 @@ $ echo foo > foo
 $ dvc add foo
 $ ls -a
 
-.dvc .git foo foo.dvc
+.dvc .git code.py foo foo.dvc
 
 $ dvc destroy
 
 This will destroy all information about your pipelines, all data files, as well as cache in .dvc/cache.
-Are you sure you want to continue? [y/n]
+Are you sure you want to continue?
 yes
 
 $ ls -a
 
-.git .gitignore foo
+.git code.py foo
 ```
 
-Let's have a look what happens when the `cache` directory is set to another
-location.
+#### Let's have a look what happens when the `cache` directory is set to another location.
```

Comment:
It should be a title, not a full sentence

Reply:
This `code.py` can be removed.